### ISSUE TITLE
other_block broken for chests in 1.14.3

### ISSUE THREAD

**Issue by mergu** [DenizenScript/Denizen-For-Bukkit#1987]
Haven't tested in any other versions, `l@location.other_block` fails for chests/trapped chests in 1.14.3. Works fine for doors/beds.

![image](https://user-images.githubusercontent.com/29823405/60316590-2be31a00-9939-11e9-963e-54a8fec5a5d2.png)

```
> version
[00:12:03 INFO]: This server is running Paper version git-Paper-115 (MC: 1.14.3) (Implementing API version 1.14.3-R0.1-SNAPSHOT)
[00:12:03 INFO]: Checking version, please wait...
[00:12:04 INFO]: You are running the latest version
> version denizen
[00:12:11 INFO]: Denizen version 1.0.6-SNAPSHOT (build 898-DEV)
[00:12:11 INFO]: Author: The DenizenScript Team
```

**Comment by mcmonkey4eva** [comment]
The code inside this tag is a bit hacked-in, using pre-1.13 BlockStates as a basis. Probably needs to be rewritten to use BlockData.

**Comment by mcmonkey4eva** [comment]
Actually that output would appear to be this check failing:
```java
                InventoryHolder holder = ((Chest) state).getBlockInventory().getHolder();
                if (holder instanceof DoubleChest) {
```
Which... not sure why it's failing but that code is a bit of a jank way of doing things.
It can indeed be modernized/replaced using https://hub.spigotmc.org/javadocs/spigot/org/bukkit/block/data/type/Chest.html which is imperfect but gives sufficient data to redesign chest support in the tag.

**Comment by mcmonkey4eva** [comment]
Fixed using blockdata as mentioned above for 1.13/1.14
Testing shows it works fine now in 1.14.3 with common chest configurations.
Thanks for the report!

**Comment by mergu** [comment]
Thanks for the quick fix!
